docs/reference/esql/functions/examples/bucket.asciidoc

```diff
@@ -58,6 +76,20 @@ include::{esql-specs}/bucket.csv-spec[tag=docsBucketNumeric-result]
 Unlike the earlier example that intentionally filters on a date range, you rarely want to filter on a numeric range.
 You have to find the `min` and `max` separately. {esql} doesn't yet have an easy way to do that automatically.
 
+If the desired bucket size is known in advance, simply provide it as the second
+argument, leaving the range out:
```

Comment:
nit: The repetition doesn't read so well.
```suggestion
As for dates, one can omit the range if the desired bucket size is known in advance. Simply provide it as the second
argument:
```

Reply:
Applied (left out the dates reference, though).